CI failed but it was NOT related to this PR
```cpp
ERROR: test_simple_net_hybrid_strategy (__main__.TestSemiAutoParallelLlamaDataLoader)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/workspace/Paddle/test/collective/test_communication_api_base.py", line 79, in run_test_case
    self._launcher = subprocess.run(
  File "/usr/lib/python3.10/subprocess.py", line 526, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/usr/bin/python', '-u', '-m', 'paddle.distributed.launch', '--log_dir', '/tmp/tmpkx_sv8w9', '--devices', '0,1,2,3,4,5,6,7', 'semi_auto_llama_dataloader.py']' returned non-zero exit status 1.
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/workspace/Paddle/test/auto_parallel/hybrid_strategy/test_semi_auto_parallel_llama_model.py", line 224, in test_simple_net_hybrid_strategy
    self.run_test_case(
  File "/workspace/Paddle/test/collective/test_communication_api_base.py", line 90, in run_test_case
    raise RuntimeError(
RuntimeError: Error occurs when running this test case. The return code of command ['/usr/bin/python', '-u', '-m', 'paddle.distributed.launch', '--log_dir', '/tmp/tmpkx_sv8w9', '--devices', '0,1,2,3,4,5,6,7', 'semi_auto_llama_dataloader.py'] is 1
----------------------------------------------------------------------
Ran 9 tests in 444.641s
FAILED (errors=1)
```

--- a/python/paddle/distributed/communication/stream/all_to_all.py
+++ b/python/paddle/distributed/communication/stream/all_to_all.py
@@ -27,10 +27,10 @@ def _all_to_all_tensor_in_dygraph(
 ):
     if use_calc_stream:
         return group.process_group.all_to_all_tensor_on_calc_stream(
-            in_tensor, out_tensor
+            out_tensor, in_tensor
         )
 
-    task = group.process_group.all_to_all_tensor(in_tensor, out_tensor, sync_op)
+    task = group.process_group.all_to_all_tensor(out_tensor, in_tensor, sync_op)
     if sync_op:
         task.wait()
 

